src/components/Product.tsx

```diff
@@ -5,116 +5,116 @@ interface ProductProps {}
 
 export const Product: React.FC<ProductProps> = (props) => {
   return (
-    <Row>
-      <CheckContainer>
+    <ProductContainer>
+      <div>
         <input type="checkbox" />
-      </CheckContainer>
-      <ImageColumn>
-        <ProductImage />
-        <ButtonRow>
-          <Button>
-            <ButtonImage src={process.env.PUBLIC_URL + '/decrement.svg'} />
-          </Button>
-          <Count>
-            <CountText>1</CountText>
-          </Count>
-          <Button>
-            <ButtonImage src={process.env.PUBLIC_URL + '/increment.svg'} />
-          </Button>
-        </ButtonRow>
-      </ImageColumn>
-      <ProdcutInformationContainer>
-        <Row>
-          <ProuductTitle>
+      </div>
+      <div className="product-flex-column product-image-container">
+        <img className="product-main-image" />
```

Comment:
`product-flex-column`과 같이 css효과로 구별해줄때는 `product-image-container`를 먼저쓰고 
`product-flex-column`으로 쓰는게 더 보기 편할 것 같아요
```suggestion
      <div className="product-image-container product-flex-column">
        <img className="product-main-image" />
```